### ISSUE TITLE
The CF policies for deployment users are not allowing ECR uploads

### ISSUE THREAD

**Issue by maciekpapiez** [headline-1/alpha-k8s#3]
Fixed policy looks like this:

```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Action": [
                "s3:GetBucketLocation",
                "s3:ListAllMyBuckets"
            ],
            "Resource": "*",
            "Effect": "Allow",
            "Sid": "S3Info"
        },
        {
            "Action": [
                "eks:DescribeCluster",
                "eks:ListClusters"
            ],
            "Resource": "*",
            "Effect": "Allow",
            "Sid": "ClusterInfo"
        },
        {
            "Action": [
                "ecr:GetAuthorizationToken",
                "ecr:DescribeRepositories",
                "ecr:CreateRepository"
            ],
            "Resource": "*",
            "Effect": "Allow",
            "Sid": "RepositoryInfoAndCreation"
        },
        {
            "Action": "ecr:*",
            "Resource": "arn:aws:ecr:eu-west-1:*:repository/REPO_NAME/*",
            "Effect": "Allow",
            "Sid": "RepositoryAllAccess"
        }
    ]
}
```

**Comment by krzysztof-miemiec** [comment]
The only difference I spotted is between:
```
        {
            "Action": "ecr:*",
            "Resource": "arn:aws:ecr:eu-west-1:*:repository/REPO_NAME/*",
            "Effect": "Allow",
            "Sid": "RepositoryAllAccess"
        }
```
and
```
{
  "Sid":"RepositoryAllAccess",
  "Effect":"Allow",
  "Action":"*",
  "Resource": { "Fn::Join":["", ["arn:aws:ecr:::repository/", { "Ref":"NamespaceName" }, "/*"]] }
}
```

Does it mean that the action has to be prefixed with `ecr:`?

**Comment by maciekpapiez** [comment]
eu-west-1 did the trick, I think.

**Comment by krzysztof-miemiec** [comment]
I just checked that part with AWS Policy Simulator and yes, `arn:aws:ecr:*:*:repository/` instead of `arn:aws:ecr:::repository/` will do the job 👍

**Comment by krzysztof-miemiec** [comment]
Fixed in [v0.1.2](https://github.com/headline-1/k8s-alpha-plugin/releases/tag/v0.1.2)
